sympy/functions/elementary/exponential.py

```diff
@@ -578,8 +590,9 @@ def eval(cls, arg, base=None):
             return arg._eval_func(cls)
 
         if arg.is_number:
+            I = S.ImaginaryUnit
```

Comment:
If you make some more changes (seems like a waste to run the tests again etc for this small comment), but it may make sense to define this just before the `if arg.is_number` and then use it even in the lines before it is redefined (and not redefine it).

Reply:
What do you think, @gschintgen ? Are there more changes? Everything except for this looks fine to me. This detail could be handled in some other PR with other changes.